test/library/standard/Map/testMapIO.future

```diff
@@ -0,0 +1,4 @@
+bug: cannot read a map from its written format
```

Comment:
For posterity, I think it'd be helpful to call out you're adding this future in your commit message :)

Reply:
I've opened #20004 and force-pushed a change to this future and its relevant files to make things a bit cleaner.